prow/cluster/deck_deployment.yaml

```diff
@@ -33,13 +33,14 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: deck
-        image: gcr.io/k8s-prow/deck:0.55
+        image: gcr.io/k8s-prow/deck:0.56
         imagePullPolicy: Always
         ports:
           - name: http
             containerPort: 8080
         args:
         - --build-cluster=/etc/cluster/cluster
+        - --tide-url=http://tide/
```

Comment:
I don't think this is quite right cc @cjwagner

Reply:
Did you deploy the tide service?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/cluster/tide_service.yaml

Reply:
No, I bumped / deployed the services that consume the podspec as @cjwagner listed, I didn't expect to need to deploy anything for tide. Perhaps this should have had an announcement?

Reply:
Yes, it should.